### DISCUSSION TITLE
Omnisend Support and Delisting

### DISCUSSION THREAD

**Discussion by dannyleungkc** [swellstores/community#84] [Help] [answered]
Hi there,

A while back ago, I reported some bugs on Order Confirmation and Abandoned Cart Email template to both Omnisend and Swell. Product link on email template is incorrect and product images are not showing in the emails etc.

Staff from Omnisend have tried to contact Swell to resolve the issues multiple times but there's no responses from Swell. I got an update from Omnisend and due to the frustration they have. They have delisted Swell integration on their website...

Understand you guys might be very busy or shortstaffed but it would be great if you could at least respond them. Now Swell is losing an integration as a result... I thought I'd flag it here...

<img width="1164" alt="Screenshot 2023-01-12 at 16 44 47" src="https://user-images.githubusercontent.com/119118081/212021296-76347a15-64e5-4ff3-b889-ee4f97ba037d.png">

**Comment by rodrigoddalmeida** [comment, marked as answer]
Hi @dannyleungkc, thank you for flagging this. We have internally reached out to Omnisend about reported bugs, as we are actively tracking them. This integration is still definitely one of our supported features, and a number of merchants do heavily use it.

We'll keep this going, bugfixes and updates to our integrations should be posted on our [Changelog](https://www.swell.is/changelog) and newsletters.

**Reply by mewa** [reply]
Is it still supported?

Whenever I try connecting Omnisend (in Swell's dashboard) I get this error instead:
![2023-04-28-132635_541x131_scrot](https://user-images.githubusercontent.com/7347194/235135694-0f780943-043c-46cd-bc78-7992f716f690.png)

Using the same API key directly with Omnisend API is successful.

**Reply by ericingram** [reply]
It should be supported but there could have been breaking changes to the Omnisend API.

@StevenSwell can you look into this

**Reply by steven-k1** [reply]
@mewa Will have this looked into and follow up as updates become available

**Reply by logeshswell** [reply]
@mewa It's a bug at our end and we will create a bug ticket internally and fix it at the earliest.

**Reply by mewa** [reply]
Looking forward to it :pray:

**Reply by logeshswell** [reply]
@mewa This bug is fixed and deployed at our end, can you check and confirm?

**Reply by mewa** [reply]
I can confirm that I managed to enable the integration and there's a validation customer record visible in my Omnisend account.

However, after enabling the integration it just says it's syncing and runs GET `/admin/api/data/settings/integrations/services/omnisend` every few seconds, indefinitely (background job?).
![omnisend](https://github.com/swellstores/community/assets/7347194/4f67205b-d01a-4511-8b55-1915d273d73d)
No new records are available in Omnisend either (this includes both existing customers as well as when I added a test customer after the integration has been enabled).

**Reply by logeshswell** [reply]
Yes @mewa I also noticed the same issue. It will be fixed and rolled out very shortly.

**Reply by logeshswell** [reply]
@mewa This sync issue is fixed and deployed yesterday. It is working fine as expected now. Can you also check and confirm?

**Reply by mewa** [reply]
Thank you, it does seem to work fine now!

**Reply by mewa** [reply]
@logeshswell one thing I noticed – some customers choose opt out of receiving email marketing, but that information is not propagated to Omnisend (thus risking sending emails to them).
